src/Core/EmailService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace VPlugins\MalwareScanner\Core;
+
+class EmailService {
+
+    /**
+     * Send scan results via email.
+     *
+     * @param array $report The scan report.
+     * @return bool True if the email was sent successfully, false otherwise.
+     */
+    public static function send_scan_email($report) {
+        // Retrieve the notification emails from settings
+        $to = get_option('malware_scanner_notification_email', get_option('admin_email'));
+    
+        // Split emails into an array and trim whitespace
+        $recipients = array_map('trim', explode(',', $to));
+    
+        // Email subject
+        $subject = 'Malware Scan Report - ' . get_bloginfo('name');
+    
+        // Email body
+        $message = '<h1>Malware Scan Results</h1>';
+        $message .= '<p><strong>Malware Detected:</strong> ' . esc_html($report['malwareDetected']) . '</p>';
+        $message .= '<p>For more details, check your WordPress dashboard.</p>';
+    
```

Comment:
add `_()` or `_e()` function for translatable string

Reply:
Updated